packages/protocol/contracts/common/Accounts.sol

```diff
@@ -129,7 +135,10 @@ contract Accounts is
    * @param name The name to set.
    */
   function setName(string memory name) public {
-    require(isAccount(msg.sender), "Unknown account");
+    require(
+      isAccount(msg.sender),
+      string(abi.encodePacked(msg.sender, " is not a registered account"))
+    );
```

Comment:
should we make this a modifier to DRY up the code a bit?
https://docs.soliditylang.org/en/v0.6.2/contracts.html#function-modifiers
not sure how the compiler optimizes but the string might also be redundant in the bytecode

Reply:
@yorhodes thank you for pointing this out. I will make this change.